publication-commons/src/test/java/no/unit/nva/publication/service/impl/FailingDynamoClient.java

```diff
@@ -0,0 +1,571 @@
+package no.unit.nva.publication.service.impl;
```

Comment:
This class does nothing different except for calling the wrapped client for all methods except for the batchWriteItem